test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,16 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+
+        isAnyNull("abc", 555, null);
+        isAnyNull((Object) null);
+        isAnyNull(Integer.valueOf(-123), 1, null);
+        isAnyNull(null, null, null);
+
```

Comment:
Could add a few comments as to what is being tested in these separate logical units of code in this block.